src/main/java/http/MIME.java

```diff
@@ -0,0 +1,42 @@
+package http;
+
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
+public enum MIME {
+	HTML("text/html", "html"),
+	CSS("text/css", "css"),
+	JS("application/javascript", "js"),
+	ICO("image/x-icon", "ico"),
+	PNG("image/png", "png"),
+	JPG("image/jpeg", "jpg"),
+	TXT("text/plain", "txt"),
+	EOT("font/EOT", "eot"),
+	SVG("font/SVG", "svg"),
+	WOFF("font/WOFF", "woff"),
+	WOFF2("font/WOFF2", "woff2"),
+	TTF("font/ttf", "ttf");
+
+	public final String contentType;
+	public final String extension;
+
+	MIME(String contentType, String extension) {
+		this.contentType = contentType;
+		this.extension = extension;
+	}
+
+	private String getExtension() {
+		return extension;
+	}
+
+	public boolean isTemplate() {
+		return this.equals(MIME.HTML) || this.equals(MIME.TXT);
+	}
+
+	public static MIME findMimeFromExtension(String extension) {
```

Comment:
`MIME.xxx`와 같이 MIME 이 prefix로 붙은 꼴이되기 때문에  메서드 네이밍에 꼭 enum class 명이들어가지는 않아도 괜찮을 것 같아요. 🙂

```suggestion
	public static MIME findByExtension(String extension) {
```